utest/run.py

```diff
@@ -48,55 +57,47 @@ def get_tests(directory=None):
         elif testfile.match(name):
             modname = os.path.splitext(name)[0]
             if modname in imported:
-                print("Test module '%s' imported both as '%s' and '%s'. "
-                      "Rename one or fix test discovery."
-                      % (modname, imported[modname],
-                         os.path.join(directory, name)), file=sys.stderr)
+                print(
+                    f"Test module '{modname}' imported both as '{imported[modname]}' and "
+                    + "'{os.path.join(directory, name)}'. Rename one or fix test discovery.",
+                    file=sys.stderr,
+                )
                 sys.exit(1)
             module = __import__(modname)
             imported[modname] = module.__file__
             tests.append(unittest.defaultTestLoader.loadTestsFromModule(module))
     return tests
 
 
-def parse_args(argv):
-    docs = False
-    verbosity = 1
-    try:
-        options, args = getopt.getopt(argv, 'hH?vqd',
-                                      ['help', 'verbose', 'quiet', 'doc'])
-        if args:
-            raise getopt.error('no arguments accepted, got %s' % list(args))
-    except getopt.error as err:
-        usage_exit(err)
-    for opt, value in options:
-        if opt in ('-h', '-H', '-?', '--help'):
-            usage_exit()
-        if opt in ('-q', '--quiet'):
-            verbosity = 0
-        if opt in ('-v', '--verbose'):
-            verbosity = 2
-        if opt in ('-d', '--doc'):
-            docs = True
-            verbosity = 2
-    return docs, verbosity
-
-
 def usage_exit(msg=None):
     print(__doc__)
     if msg is None:
         rc = 251
     else:
-        print('\nError:', msg)
+        print("\nError:", msg)
         rc = 252
     sys.exit(rc)
 
 
-if __name__ == '__main__':
-    docs, vrbst = parse_args(sys.argv[1:])
-    tests = get_tests()
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(add_help=False, description=__doc__)
+    parser.add_argument("-I", "--interpreter", default=sys.executable)
+    parser.add_argument("-h", "--help", action="store_true")
+    parser.add_argument("-q", dest="vrbst", action="store_const", const=0)
+    parser.add_argument("-v", dest="vrbst", action="store_const", const=2)
+    parser.add_argument("-d", dest="docs", action="store_true")
```

Comment:
`-q`, `-v` and `-d` should get long options like they had earlier. `--quiet` being missing is the reason running unit tests as part of acceptance tests fails.